index.html

```diff
@@ -28,17 +28,19 @@
                     company:    "DuraSpace",
                     companyURL: "http://duraspace.org/"}
           ],
-          edDraftURI: "https://fcrepo.github.io/fcrepo-specification/",
+          publishDate: "2021-05-01",
```

Comment:
Were you building in time for us to actually update https://fedora.info ?

Reply:
That and uncertain timeline for review... but since everyone jumped on it we could either change the date or just work towards it for publication on fedora.info. I have assumed you will be able to work to get it up on  https://fedora.info ?

Reply:
Sounds good. I will schedule publishing on Saturday, May 1st.